package.json

```diff
@@ -186,7 +186,7 @@
     "denymount": "^2.3.0",
     "docker-modem": "^2.0.4",
     "docker-progress": "^4.0.0",
-    "docker-qemu-transpose": "^1.0.2",
+    "docker-qemu-transpose": "^1.0.3-27-specified-dockerfile-bde92fe1c1252f71aebb1e1df91dc0395ecf3cf1",
```

Comment:
This will be changed to `^1.0.3` after depends-on PR balena-io-modules/docker-qemu-transpose/pull/28 is merged and released.